scripts/scil_assign_color_to_tractogram.py

```diff
@@ -43,6 +44,17 @@ def _build_arg_parser():
     p1.add_argument('--dict_colors',
                     help='Dictionnary mapping basename to color.\n'
                          'Same convention as --fill_color.')
+    p1.add_argument('--use_dps', metavar='DPS_KEY',
+                    help='Use the data_per_streamline (scalar) for coloring, '
+                         'linear from min to max.')
+    p1.add_argument('--use_dpp', metavar='DPP_KEY',
+                    help='Use the data_per_point (scalar) for coloring, '
+                         'linear from min to max.')
```

Comment:
Except commit_weights at the moment we have no other example. 
Nothing in scilpy actually save very important data in a consistent manner in the data_per_*.
Even the afd_along simply save nifti rather than assigning the values to each streamline.